### PR TITLE
feat: add sweertAlert for createSpace and call auth service FIX #4

### LINKED ISSUE #4: Adding Actual API call
Replace with actual API call to create a space and how popup with [SweetAlert2](SWeetalert2.github.io/) displaying generated space URL (it will be in the response) :

https://github.com/Rater-Inc/frontend-app/blob/081bd17ac12b933207e5bd6fbbcb1da0753bf3e9/src/pages/CreatePage/SpaceDetails.jsx#L46-L63

Replace with the real API 
https://github.com/Rater-Inc/frontend-app/blob/081bd17ac12b933207e5bd6fbbcb1da0753bf3e9/src/pages/RatePage/RatingPage.jsx#L59-L66

Add the api call based on space id; fetch player and metric list and render page with them. DO NOT forget to check `authentication` status.
https://github.com/Rater-Inc/frontend-app/blob/081bd17ac12b933207e5bd6fbbcb1da0753bf3e9/src/pages/RatePage/RatingPage.jsx#L52-L57

Replace with submit ratings API and show popup with [SweetAlert2](SWeetalert2.github.io/):
https://github.com/Rater-Inc/frontend-app/blob/081bd17ac12b933207e5bd6fbbcb1da0753bf3e9/src/pages/RatePage/RatingPage.jsx#L78-L86

### PR DESCRIPTION
**Pull request by burakkyildirim11**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ function App() {
       <CssBaseline />
       <Routes>
         <Route path="/" element={<CreateSpacePage />} />
-        <Route path="/space/:spaceId" element={<RatingPage />} />
+        <Route path="/rating-page/:spaceId" element={<RatingPage />} />
```
Comment (fehmisener) [inline src/App.jsx:20]: Should be `rating`

File: src/pages/CreatePage/SpaceDetails.jsx [hunk 2]
```diff
@@ -54,10 +58,21 @@ const SpaceDetails = ({ metrics, players }) => {
...
       .then((response) => response.json())
       .then((data) => {
+        Swal.fire({
+          title: "Success",
+          text: "Space Created Successfully!",
+          icon: "success"
+        });
         console.log('Space created successfully:', data);
+
+        navigate(`/rating-page/${data.spaceId}`, { state: { link: data.link} });
```
Comment (fehmisener) [inline src/pages/CreatePage/SpaceDetails.jsx:68]: Do not navigate directly, navigate with alert confirmation button

File: src/pages/CreatePage/SpaceDetails.jsx [hunk 3]
```diff
@@ -45,7 +49,7 @@ const SpaceDetails = ({ metrics, players }) => {
 
     // Replace with actual API call
 
-    fetch('https://localhost:7242/api/Space/CreateSpace', {
+    fetch('http://localhost:8031/api/Space/CreateSpace', {
```
Comment (fehmisener) [inline src/pages/CreatePage/SpaceDetails.jsx:52]: Move this API call to the api class - F1

File: src/pages/RatePage/RatingPage.jsx [hunk 4]
```diff
@@ -57,12 +59,24 @@ const RatingPage = ({}) => {
   }, [spaceId]);
 
   const handleLogin = () => {
-    // Dummy authentication for demonstration
-    if (password === '123') {
-      setAuthenticated(true);
-    } else {
-      setError('Invalid password');
-    }
+    const url = `http://localhost:8031/api/Auth?link=${encodeURIComponent(link)}&password=${encodeURIComponent(password)}`;
```
Comment (fehmisener) [inline src/pages/RatePage/RatingPage.jsx:62]: Check F1

**Review by fehmisener — CHANGES_REQUESTED** [review]

**Comment by fehmisener** [comment]
Don't forget to include a issue number in your commit

File: src/pages/CreatePage/SpaceDetails.jsx (hunk 3 above)
Comment (burakkyildirim11) [reply src/pages/CreatePage/SpaceDetails.jsx:52]: Will be fixed in other PR

File: src/pages/RatePage/RatingPage.jsx (hunk 4 above)
Comment (burakkyildirim11) [reply src/pages/RatePage/RatingPage.jsx:62]: Will be fixed in other PR

File: src/pages/CreatePage/SpaceDetails.jsx [hunk 5]
```diff
@@ -62,10 +62,12 @@ const SpaceDetails = ({ metrics, players }) => {
           title: "Success",
           text: "Space Created Successfully!",
           icon: "success"
+        }).then((result) => {
+          if (result.isConfirmed) {
+            navigate(`/rating/${data.spaceId}`, { state: { link: data.link } });
+          }
         });
         console.log('Space created successfully:', data);
```
Comment (fehmisener) [inline src/pages/CreatePage/SpaceDetails.jsx:70]: We should remove this unnecessary logs later :)

**Review by fehmisener — APPROVED** [review]
LGTM @burakkyildirim11

**Pull request merged**